skyvern/services/otp_service.py

```
@@ -145,14 +147,25 @@ async def _get_otp_value_from_url(
+    if not content:
+        return None
 
-    magic_link = json_resp.get("magic_link", None)
-    if magic_link:
-        return OTPValue(value=magic_link, type=OTPType.MAGIC_LINK)
-    return None
+    otp_value: OTPValue | None = OTPValue(value=content, type=OTPType.TOTP)
+    if isinstance(content, str) and len(content) > 10:
+        try:
+            otp_value = await parse_otp_login(content, organization_id)
+        except Exception:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)